### ISSUE TITLE
Terminating at Epoch 9

### ISSUE THREAD

**Issue by wz337** [mesnico/RelationNetworks-CLEVR#1]
Your implementation is really great. I am interested in the model and running experiment on it. However, I have tried on two different machines and both of them are terminated at Epoch 9. Do you have the same problem?

**Comment by fabiocarrara** [comment]
Are there any error messages when the process terminates?
Could you provide the invocation of the training script you used and the last lines of the output?

**Comment by wz337** [comment]
Thanks for the quick reply. I started a screen session running the experiment and left it overnight. When I checked this morning, the screen was terminated so I couldn't see any error message. When I rerun it again, it was giving me cuda out of memory error but tops says my process is running while the GPUs are free (but couldn't allocate anything and rebooting the machine wasn't helping). Therefore, I am wondering whether you have any idea of what the issue might be.
